include/fuzzuf/executor/native_linux_executor.hpp

```diff
@@ -38,42 +40,24 @@
 // Responsibility (TODO):
 //  - The lifetime for the class itself and the time period of validity of member variables must match (TODO because checks have not completed)
 //      - This is to ensure the robustness
-class NativeLinuxExecutor : public Executor {
+class NativeLinuxExecutor : public Executor, public AFLEdgeCoverage, public FuzzufBBCoverage {
```

Comment:
Do these classes really have to be inherited from?  I feel Executor is not one kind of "something that creates and manages shared memory". Rather, "something that creates and manages shared memory" would be naturally a component constituting Executor, and thus Class Composition can be more appropriate.

Reply:
Fixed with [8a1bff5](https://github.com/fuzzuf/fuzzuf/pull/34/commits/8a1bff54f87568299cf3cf66b8237d630c5f2804).

Reply:
Nice.